### DISCUSSION TITLE
Welcome to WGCNA-shinyApp Discussions!

### DISCUSSION THREAD

**Discussion by ShawnWx2019** [ShawnWx2019/WGCNA-shinyApp#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by veritas515nature** [comment]
为啥用不了呢，也重装不上

**Reply by veritas515nature** [reply]
重装最新版tbtools也不行

**Reply by ShawnWx2019** [reply]
在安装该插件之前，Rserver.plugin安装成功了吗？

**Reply by veritas515nature** [reply]
> 在安装该插件之前，Rserver.plugin安装成功了吗？

把Rsever.plugin和WGCNA插件都remove重新按顺序安装，也是同样的报错，start后，报错并弹出finished窗口
![image](https://github.com/user-attachments/assets/9ec1134f-b2d2-491b-8f35-7cc531fe27f7)
![image](https://github.com/user-attachments/assets/aa1ba44c-28cb-409b-ba9c-73c6dbb31e56)

**Reply by veritas515nature** [reply]
![image](https://github.com/user-attachments/assets/944d7235-641d-47d1-b0b0-db352292cfc0)
彻底卸载tbtools重新安装最新版也不行啊

**Comment by veritas515nature** [comment]
![image](https://github.com/user-attachments/assets/d80b8e6b-c48c-49e5-a305-eede8a8ab30b)
![image](https://github.com/user-attachments/assets/a4ae20ae-f10e-414b-968f-61f7d63b3bbd)

**Reply by veritas515nature** [reply]
![image](https://github.com/user-attachments/assets/97af13d6-643e-4184-93a4-9a58f77fc235)
应该是TBTOOLS的问题，其他R插件都有类似bug

**Reply by ShawnWx2019** [reply]
好的，感谢 应该是windows版本的吧？我测试下然后向TBtools社区反馈下，

**Reply by veritas515nature** [reply]
是win版，win11，感谢

> 好的，感谢 应该是windows版本的吧？我测试下然后向TBtools社区反馈下，
